src/client/containers/formComponent/formComponent.js

```diff
@@ -2,58 +2,45 @@ import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import FormField from '../../components/formField/formField';
+import RenderFormFields from '../../components/renderFormFields/renderFormFields';
 import postUserData from '../../actions/postUserDataAction';
-import getUserData from '../../actions/getUserDataAction';
 import './formComponent.css';
 
 class FormComponent extends Component {
     state = {
-      userName: '',
+      username: '',
       email: '',
       password: '',
     };
 
-
     onInputChange = ({ target: { name, value } }) => {
       this.setState({ [name]: value });
     };
 
-    onRegistratuinSubmit = (e) => {
+    onSubmit = (e) => {
       e.preventDefault();
```

Comment:
Try not to use short names in handlers events ( "e" ), it is better to make them more understandable. And, I think that onRegistrationSubmit name is more transparent name than onSubmit

Reply:
done